src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -114,7 +117,30 @@ public void setUpdateDate(Timestamp updateDate) {
     this.updateDate = updateDate;
   }
 
-  /**
+  public Integer getCountUsers() {
+    List<String> emails = null;
+    if (Objects.nonNull(data)) {
+      ArrayList<Collaborator> collabs = new ArrayList<>();
+      if (Objects.nonNull(data.getInternalCollaborators())) {
+        collabs.addAll(data.getInternalCollaborators());
+      }
+      if (Objects.nonNull(data.getLabCollaborators())) {
+        collabs.addAll(data.getLabCollaborators());
+      }
+      emails = collabs.stream().map(collaborator -> Objects.requireNonNull(collaborator).getEmail()).collect(Collectors.toList());
```

Comment:
This can be streamlined and a little more legible with some lambda cleanup:
```
      emails = collabs
          .stream()
          .filter(Objects::nonNull)
          .map(Collaborator::getEmail)
          .filter(Objects::nonNull)
          .collect(Collectors.toList());
```